examples/basic.rs

```diff
@@ -12,3 +12,10 @@ fn main() {
     assert_eq!(map.get(Key::First), Some(&42));
     assert_eq!(map.get(Key::Second), None);
 }
+
+// Execute this during testing as well.
+#[cfg(test)]
+#[test]
+fn test_main() {
+    main();
+}
```

Comment:
Examples don't need to be tested. They should be as compact as possible to showcase some basic or neat feature of the library. Think this is something people look at on GitHub to get a feel for the library. Less noise = better.